src/programmatic_policy_learning/approaches/pendulum_stupid_approach.py

```diff
@@ -0,0 +1,62 @@
+"""A simple hardcoded approach for pendulum balancing."""
+
+from typing import TypeVar, cast
+
+import gymnasium as gym
+import numpy as np
+
+from programmatic_policy_learning.approaches.base_approach import BaseApproach
+
+_ObsType = TypeVar("_ObsType")
+_ActType = TypeVar("_ActType")
+
+
+class PendulumStupidAlgorithm(BaseApproach[_ObsType, _ActType]):
```

Comment:
this approach only works for the pendulum environment, so rather than defining generic type variables, we can use the specific types for this environment, i.e.,

```python
from numpy.typing import NDArray
class PendulumStupidAlgorithm(BaseApproach[NDArray[np.float64], NDArray[np.float64]]))
```